package.json

```diff
@@ -1,13 +1,17 @@
 {
   "name": "gitops-secrets",
+  "version": "0.0.1",
   "author": "Ryan Blunden <ryan.blunden@doppler.com>",
   "description": "GitOps encrypted secrets workflow",
   "repository": {
     "type": "git",
-    "url": "git://github.com/doppler-university/gitops-secrets.git"
+    "url": "https://github.com/doppler-university/gitops-secrets.git"
   },
-  "version": "0.0.1-DEV",
   "homepage": "https://github.com/doppler-university/gitops-secrets#readme",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/axios/axios/issues"
```

Comment:
We'll probably want to change this to https://github.com/DopplerUniversity/gitops-secrets-nodejs/issues yeah?